Tests/SubprocessTests/SubprocessTests+Unix.swift

```diff
@@ -952,9 +952,64 @@ extension SubprocessUnixTests {
         }
         try FileManager.default.removeItem(at: testFilePath)
     }
+
+    @Test(.requiresBash) func testDoesNotInheritRandomFileDescriptorsByDefault() async throws {
+        // This tests makes sure POSIX_SPAWN_CLOEXEC_DEFAULT works on all platforms
+        let pipe = try FileDescriptor.ssp_pipe()
+        defer {
+            close(pipe.readEnd.rawValue)
+        }
+        let writeFd = pipe.writeEnd.rawValue
+        let result = try await pipe.writeEnd.closeAfter {
+            return try await Subprocess.run(
+                .path("/bin/bash"),
+                arguments: ["-c", "echo hello from child >&\(writeFd); echo wrote into \(writeFd), echo exit code $?"],
+                output: .string(limit: .max),
+                error: .string(limit: .max)
+            )
+        }
+
+        #expect(result.terminationStatus.isSuccess)
+        #expect(
+            result.standardOutput?.trimmingCharacters(in: .whitespacesAndNewlines) ==
+            "wrote into \(writeFd), echo exit code 1"
+        )
+        // Depending on the platform, standard error should be something like
+        // `/bin/bash: 7: Bad file descriptor
+        #expect(!result.standardError!.isEmpty)
```

Comment:
nit: avoid force unwraps:

```suggestion
        let standardError = try #require(result.standardError)
        #expect(!standardError.isEmpty)
```